documentation/sprint-80.md

```diff
@@ -0,0 +1,33 @@
+# Sprint 80 Documentation Release Notes
```

Comment:
Should we really add this in German? I'd prefer to have everything in the release notes in English - it's a bit weird IMO to have an English starting page, and German content after the link.

Reply:
ok. will change that.